src/Entity/Dataset.php

```diff
@@ -59,17 +59,17 @@ class Dataset extends Entity
     /**
      * Cold Storage Tag
      */
-    const TAG_COLDSTORAGE = 'Coldstorage';
+    const TAG_COLD_STORAGE = 'Cold Storage';
```

Comment:
Make public

Reply:
We can run them when are merging this epic into develop?